beast/tools/cut_catalogs.py

```diff
@@ -158,7 +157,7 @@ def make_cuts(cat_file, partial_overlap=False, flagged=False, flag_filter=None):
 
     # read in the catalog
     cat = Table.read(cat_file)
-    filters = [c[0:-5] for c in cat.colnames if "RATE" in c and "RATERR" not in c]
+    filters = [c[0:-5] for c in cat.colnames if "VEGA" in c]
```

Comment:
Why changing from RATE to VEGA.  This switches between fluxes to magnitudes, right?

Reply:
This line is just for figuring out the base filter name. For example, extracting "F475W" from "F475W_RATE". Thus, this if statement does not need to be any complex like "if "RATE" in c and "RATERR" not in c".

Reply:
Ok.   So, it worked before.  And you switched as you prefer to use the VEGA column, because...?

Reply:
Just trying to make sure I understand this change.

Reply:
Yes, it worked before, but the old if statement requires extra job. What I did is just to simplify the if statement.

Reply:
When you do `filters = [c[0:-5] for c in cat.colnames if "RATE" in c]`, it returns:
['F110W',
 'F110W_R',
 'F160W',
 'F160W_R',
 'F275W',
 'F275W_R',
 'F336W',
 'F336W_R',
 'F475W',
 'F475W_R',
 'F814W',
 'F814W_R',
 'F127M',
 'F127M_R',
 'F139M',
 'F139M_R',
 'F153M',
 'F153M_R']
 
 To exclude items with '_R', the old if statement added  this extra condition, `and "RATERR" not in c`. We don't need to do this to figure out the filter list.

Reply:
Yes.  Will do so now.